codemod_tox/conditional.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Optional
+
+from .base import ToxBase
+from .env import ToxEnv
+from .parse import TOX_CONDITIONAL_RE
+
+
+@dataclass(frozen=True)
+class ToxConditional(ToxBase):
+    lines: tuple[tuple[Optional[ToxEnv], str], ...]
+
+    def evaluate(self, env: str) -> str:
+        buf: list[str] = []
+        for i, j in self.lines:
+            if isinstance(i, ToxEnv):
+                if i.matches(env):
+                    buf.append(j)
+            else:
+                buf.append(j)
+        return "\n".join(buf)
+
+    @classmethod
+    def parse(cls, value: str) -> "ToxConditional":
+        lines: list[tuple[Optional[ToxEnv], str]] = []
+        for line in value.splitlines():
+            print(repr(line))
```

Comment:
print()?

Reply:
sigh, will replace with a VLOG if I can figure out how to make pytest change its log capture level